src/content/hints.ts

```diff
@@ -11,40 +11,61 @@ export async function displayHints(intersectors: Intersector[]) {
 	// when there are multiple mutations or intersections happening
 	const localStorage = await browser.storage.local.get(["showHints"]);
 	const showHints = (localStorage["showHints"] as boolean) ?? true;
-	if (showHints && !hintsUpdateTriggered) {
+	if (showHints && !hintsUpdateTriggered) { // && !hintsUpdateTriggered
```

Comment:
I don't understand the comment

Reply:
Sorry this was just me adding a code comment while I removed code when trying to understand how it works. Will be removed, should have cleaned this code up a bit more to avoid confusion!